xbmc/windowing/Resolution.cpp

```diff
@@ -143,11 +143,33 @@ void CResolutionUtils::FindResolutionFromWhitelist(float fps, int width, int hei
     }
   }
 
+  CLog::Log(LOGDEBUG, "No double whitelisted resolution matched, trying 3:2 pullback");
+
+  for (const auto &mode : indexList)
```

Comment:
left align `&`, https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types